5_recognition.ipynb

```diff
@@ -430,60 +775,109 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "`lego`は、絶対座標上で約（x, y）=（0.6, 0.0）の座標に置かれているようです。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "`lego` seems to be placed at about (x, y) = (0.6, 0.0) position on absolute coordinates."
```

Comment:
"`lego` seems to be placed at about (x, y) = (0.6, 0.0) position in absolute coordinates."